Formula/bazarr.rb

```diff
@@ -0,0 +1,43 @@
+class Bazarr < Formula
+  desc "Companion to Sonarr and Radarr for managing and downloading subtitles"
+  homepage "https://www.bazarr.media"
+  url "https://github.com/morpheus65535/bazarr/releases/download/v1.0.3/bazarr.zip"
+  sha256 "540a5c4d80110ba35a566c17829cd6f31444db7ade7c6b9d0603eccaaed8a628"
+  license "GPL-3.0-only"
+  head "https://github.com/morpheus65535/bazarr.git", branch: "master"
+
+  depends_on "gcc"
+  depends_on "numpy"
+  depends_on "python@3.9"
+
+  uses_from_macos "zlib"
+
+  def install
+    libexec.install Dir["*"]
+    (libexec/"data").install_symlink pkgetc => "config"
+
+    (bin/"bazarr").write_env_script Formula["python@3.9"].opt_bin/"python3", "#{libexec}/bazarr.py", []
+  end
+
+  def post_install
+    pkgetc.mkpath
+  end
+
+  def caveats
+    <<~EOS
+      Configuration files: #{pkgetc}
+    EOS
+  end
+
+  plist_options startup: true
+  service do
+    run opt_bin/"bazarr"
+    keep_alive true
+    log_path var/"log/bazarr.log"
+    error_log_path var/"log/bazarr.log"
+  end
+
+  test do
+    system "#{bin}/bazarr", "--help"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Not really feasible, I'm afraid, given it's an interactive service/daemon (and its operation is complex and highly dependent on configuration, on top of that).

Reply:
Can you not spawn a pty to interact with whatever UI it has? You can assume that there is no pre-existing configuration in the test environment.

Reply:
@carlocab It has a web UI. So maybe what we can do is spawn an instance of `bazarr` in the background, wait a short time, then make a HTTP request to localhost on the default port? Then check the response is HTTP 200, and perhaps grep it for `<title>Bazarr</title>`? Let me know if that sounds alright, but moreover, if there's an existing formula that does something like this, I would be grateful to be pointed to it so I can mimic that.

Reply:
That sounds good.

Reply:
@carlocab Great. Any example of an existing formula test doing something like this? It's probably beyond my Ruby abilities, so I could definitely use something to copy.

Reply:
Can't think of a specific example off the top of my head, but there should be many. Try looking at formulae that do `fork do` and then `curl` in their `test` block.

Reply:
Cheers

Reply:
Hopefully the test looks good now.